IdentityCore/src/cache/MSIDCacheAccessor.h

```diff
@@ -68,6 +70,12 @@
                                          context:(id<MSIDRequestContext>)context
                                            error:(NSError **)error;
 
+- (MSIDPrimaryRefreshToken *)getPrimaryRefreshTokenWithAccount:(MSIDAccountIdentifier *)account
```

Comment:
nit: can we simplify how we call PrimaryRefrestToken over and over? 
Perhaps PRT? or even PrimaryRT?